airbrakes/mock/mock_imu.py

```diff
@@ -61,22 +65,40 @@ def __init__(
         # Starts the process that fetches data from the log file
         self._data_fetch_process = multiprocessing.Process(
             target=self._fetch_data_loop,
-            args=(
-                self._log_file_path,
-                real_time_simulation,
-                start_after_log_buffer,
-            ),
+            args=(self._log_file_path, real_time_simulation, start_after_log_buffer),
             name="Mock IMU Process",
         )
 
         # Makes a boolean value that is shared between processes
         self._running = multiprocessing.Value("b", False)
 
     @staticmethod
-    def _convert_invalid_fields(value):
-        if not value:
-            return None
-        return ast.literal_eval(value)  # Convert string to list
+    def _convert_invalid_fields(value) -> list:
+        """
+        Convert invalid fields to Python objects or None.
+        :param value: The value to convert.
+        :return: The converted value
+        """
```

Comment:
the comment which said that this is used for converting the string to list should be kept imo